pkg/storaged/fsys-tab.jsx

```diff
@@ -95,18 +95,54 @@ export function find_blocks_for_mount_point(client, mount_point, self) {
     return blocks;
 }
 
+function find_children_for_mount_point(client, mount_point, self) {
+    const children = {};
+
+    function is_self(b) {
+        return self && (b == self || client.blocks[b.CryptoBackingDevice] == self);
+    }
+
+    for (const p in client.blocks) {
+        const b = client.blocks[p];
+        const fs = client.blocks_fsys[p];
+
+        if (is_self(b))
+            continue;
+
+        if (fs) {
+            for (const mp of fs.MountPoints) {
+                const mpd = utils.decode_filename(mp);
+                if (mpd.length > mount_point.length && mpd.indexOf(mount_point) == 0 && mpd[mount_point.length] == "/")
+                    children[mpd] = b;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19388-20230925-134144-be2f6638-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)